bitnami/postgresql-repmgr/10/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -237,6 +252,60 @@ repmgr_get_primary_node() {
     echo "$primary_port"
 }
 
+
+
+########################
+# switchover standby after long primary wait
+# Globals:
+#   REPMGR_*
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+
+repmgr_switchover_standby() {
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME:-60}"
+
+    primary_host=""
+    primary_port="${REPMGR_PORT_NUMBER}"
+
+    info "Switchover from standby to primary . After unavailable primary for '$timeout' seconds . Assuming the primary role..."
+
+    echo "promote_me" >> "$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME"
+
+    debug "Marking node to be promoted from standby to primary by writing the following file: '$REPMGR_PROMOTE_STANDBY_LOCK_FILE_NAME' "
+
+    [[ -n "$primary_host" ]] && debug "Primary node: '${primary_host}:${primary_port}'"
+    echo "$primary_host"
+    echo "$primary_port"
```

Comment:
I don't see any circumstances where $primary_host won't be empty. 

What is the purpose of the two `echo`?

Reply:
this was for verification purposes I will get rid of it